src/types/xdai/xdai.ts

```diff
@@ -82,6 +49,29 @@ export class Xdai extends Ethereum {
           },
         ];
         break;
+      case NodeClient.NETHERMIND:
+        versions = [
+          {
+            version: '1.12.3',
+            clientVersion: '1.12.3',
+            image: 'nethermind/nethermind:1.12.3',
+            dataDir: '/nethermind/nethermind_db',
+            walletDir: '/nethermind/keystore',
+            configPath: '/nethermind/configs/xdai.cfg',
+            networks: [NetworkType.MAINNET],
+            breaking: false,
+            generateRuntimeArgs(data: CryptoNodeData): string {
```

Comment:
I prefer the config to stay in the config file as much as possible. If possible, our preference would be to run it with only one flag, `--config` and have that config contain everything rather than splitting the config data between the config file and runtime flags. Sometimes runtime flags are unavoidable, but if these can be put into the config itself I would like to see them there.

Reply:
thanks, fixed